### DISCUSSION TITLE
DIA-NN data normalization

### DISCUSSION THREAD

**Discussion by akila-wijerathna-yapa** [vdemichev/DiaNN#955] [Q&A] [answered]
Hi Vadim,
My reviewer ask a question about how DIA-NN data normalization work. When analyzing multi samples (even though we are sure about injecting known amount of digested peptides) how should I perform data normalization for DIA data? 

i.e Normalize to Total peak area for each sample (like RPKM/FPKM normalization methods in RNAseq) or 

Does it happening inside the DIA-NN algorithm or should I have to do it manually in downstream pipeline?

**Comment by vdemichev** [comment, marked as answer]
Hi Akila,

The main normalisation algorithm is described in DIA-NN Nature Methods 2020 paper.
On top of that, DIA-NN also uses RT-dependent normalisation, wherein an extra normalisation factor is applied calculated as the running median of the measured fold changes (sample vs other samples) across the retention time, which corrects for effects such as loss of hydrophilic or hydrophobic peptides in specific sample compared to other samples.

All the above is performed on the precursor level and is enough for many experiments. You might want to perform on top of that also protein-level normalisation and see if it's beneficial for your experiment - this is left to the user as it's a simple task (several code lines of a script) compared to precursor-level normalisation. 

Best,
Vadim

**Reply by akila-wijerathna-yapa** [reply]
Thank you Vadim.
